ownership-chain/pallets/living-assets-ownership/src/lib.rs

```diff
@@ -42,24 +34,18 @@ pub mod pallet {
 		/// However it seems the max supported length in browsers is 2,048 characters.
 		///
 		/// The base should be capped at 2,015 characters in length. This ensures room for
-		/// the token URI formation, where it combines `BaseURILimit`, a `'/'`, and a `tokenID`
+		/// the token URI formation, where it combines `BaseURILimit`, a `'/'`, and a `AssetId`
 		/// (which takes up 33 characters).
 		#[pallet::constant]
 		type BaseURILimit: Get<u32>;
 
-		/// This associated type defines a conversion from the `AccountId` type, which is internal
-		/// to the implementing type (represented by `Self`), to an `H160` type. The `H160` type
-		/// is commonly used to represent Ethereum addresses.
-		type AccountIdToH160: Convert<Self::AccountId, H160>;
-
-		/// This associated type defines a conversion from an `H160` type back to the `AccountId`
-		/// type, which is internal to the implementing type (represented by `Self`). This
-		/// conversion is often necessary for mapping Ethereum addresses back to native account IDs.
-		type H160ToAccountId: Convert<H160, Self::AccountId>;
+		/// Simply a null address in EVM.
```

Comment:
since `AccountId` is an associated type and its bounds are defined in `frame_system` there was no way to create `null` address from `[u8;20]`, i.e no `From<[u8;20]>` trait impl. This is why I am passing the null address value from runtime.